crates/tabby-common/src/registry.rs

```diff
@@ -54,6 +71,18 @@ pub struct ModelRegistry {
     pub models: Vec<ModelInfo>,
 }
 
+lazy_static! {
+    pub static ref GGML_MODEL_RELATIVE_PATH: String =
+        format!("ggml{}model.gguf", std::path::MAIN_SEPARATOR_STR);
```

Comment:
```suggestion
    pub static ref LEGACY_GGML_MODEL_PATH: String =
        format!("ggml{}model.gguf", std::path::MAIN_SEPARATOR_STR);
```

Please also update related function names, e.g migrate_relative_model_path